docs/docsite/rst/community/collection_contributors/collection_release_with_branches.rst

```diff
@@ -0,0 +1,320 @@
+.. _collection_release_with_branches:
+
+Releasing collections with release branches
+============================================
+
+Collections MUST follow the `semantic versioning <https://semver.org/>`_ rules. See :ref:`releasing_collections` for high-level details.
+
+.. contents::
+  :local:
+
+
+Release planning and announcement
+----------------------------------
+
+#. Announce your intention to release the collection in a corresponding pinned release issue/community pinboard of the collection and in the ``#ansible-community`` `Matrix/IRC channel <https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat>`_ (and other dedicated channels if they exist).
+
+#. Ensure all the other repository maintainers are informed about the time of the following release.
+
+
+Releasing major collection versions
+-------------------------------------
+
+The new version is assumed to be ``X.0.0``.
+
+1. If you are going to release the ``community.general`` and ``community.network`` collections, create new labels ``backport-X`` and ``needs_backport_to_stable_X`` in the corresponding repositories. Copy the styles and descriptions from the corresponding existing labels.
+
+2. Ensure you are in a default branch in your local fork (we use ``main`` in the following examples).
```

Comment:
2. Ensure you are in a default branch in your local fork. In the following examples below, we use the ``main`` branch.

Reply:
please i was asked to leave a comment about the parenthesis, and how there need not be a lot of them